### PR TITLE
Added revisions to Charter Amendments

### PR DESCRIPTION
**Pull request by rayraspberry**
The changes were based on Round Table Meeting discussions

From this document:
https://docs.google.com/document/d/1gskkHp6xMqoMTr2BefQnuCeTD5fkLBl6_jOksy90sR0/edit

### CODE REVIEW

**Review by Dat-Andre — APPROVED** [review]
looks good!

**Review by Dat-Andre — APPROVED** [review]
looks  good again

File: charter/DRAFT-E-DELEGATE_INTEGRITY.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 ### 1. SEPARATION OF DELEGATES AND CONTRIBUTORS
 
-    To guarantee an effective oversight of contributors, there is a need for a separation between the function of the delegates and the contributors. Department delegates must refrain from participating through Council or Department mandates and RFPs as a contractor or contributor if this position is subject to the supervision of their own department.
+    To guarantee an effective oversight of contributors, there is a need for a separation between the function of the delegates and the contributors.
+    
+    Department delegates are permitted to participate in Council or Department RFPs as contractors or contributors. However, if they are selected, they must immediately step down from their delegate role.
```
Comment (Highlander-maker) [inline charter/DRAFT-E-DELEGATE_INTEGRITY.md:9]: Agree with this. This has been the biggest problem for me personally.

**Review by Highlander-maker — COMMENTED** [review]
Overall looks good and clear.

**Pull request merged**